hibernate-orm/core/Basic/src/main/resources/META-INF/persistence.xml

```diff
@@ -2,7 +2,17 @@
 <persistence xmlns="http://java.sun.com/xml/ns/persistence" version="1.0">
     <persistence-unit name="Demo">
         <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
-        <!-- Hibernate scanning discovers all entities. -->
+
+		<!-- Define the classes -->
+		<class>org.hibernate.brmeyer.demo.entity.User</class>
+		<class>org.hibernate.brmeyer.demo.entity.Tool</class>
+		<class>org.hibernate.brmeyer.demo.entity.Skill</class>
+		<class>org.hibernate.brmeyer.demo.entity.Community</class>
+		<class>org.hibernate.brmeyer.demo.entity.Donation</class>
+		<class>org.hibernate.brmeyer.demo.entity.Comment</class>
+		<class>org.hibernate.brmeyer.demo.entity.ServiceEvent</class>
+		<class>org.hibernate.brmeyer.demo.entity.Project</class>
+
```

Comment:
@Naros what's the reason for this change?

Reply:
@gsmet I specifically noticed that by upgrading to ORM, the `BasicJpaDemo` failed because it couldn't locate the entity classes.  Obviously there are several ways I could inject the annotated classes into the persistence unit configuration, this being just one of them.

Another solution would be to pass a list of annotated classes to the construction of the factory using `AvailableSettings#LOADED_CLASSES` much like how our test suite does.

Do you have a preferred approach?

Reply:
I don't have any preference :). I was just surprised they were automatically discovered before and that we need to declare them now.